src/API/Management/Tickets.php

```diff
@@ -2,21 +2,44 @@
 
 namespace Auth0\SDK\API\Management;
 
+use Auth0\SDK\Exception\EmptyOrInvalidParameterException;
+
 class Tickets extends GenericResource
 {
     /**
      *
      * @param  string      $user_id
      * @param  null|string $result_url
+     * @param array  $params  Array of optional parameters to add.
+     *      - client_id: Client ID of the requesting application.
```

Comment:
this `client_id` is documented as if it was a property from params, but "params" was added on this PR and I don't see any usages of client_id from it. Is this line correct?

Reply:
Updated to fix.